tests/behaviour/qbwriter.feature

```diff
@@ -646,3 +647,79 @@ Feature: Test outputting CSV-Ws with Qb flavouring.
     
     <file:/tmp/qb-id-10002.csv#csvcubed-build-activity> a prov:Activity;
     """
+
+Scenario: A pivoted shape cube should be produced as the output for the pivoted shape inputs
+  Given a pivoted shape cube with identifier "qb-id-10003" named "Pivoted Shape Cube"
+  When the cube is serialised to CSV-W
+  Then the file at "qb-id-10003.csv" should exist
+  And the file at "qb-id-10003.csv-metadata.json" should exist
+  And csvlint validation of all CSV-Ws should succeed
+  And csv2rdf on all CSV-Ws should succeed
+  And the RDF should contain
```

Comment:
Can you add an additional step which runs the QB SPARQL tests? You should be able to find examples of that above.